### ISSUE TITLE
Error install for vant 4.3.0

### ISSUE THREAD

**Issue by eivmosn** [youzan/vant#11812]
### Reproduction Link

https://codesandbox.io/s/vant-4-issue-template-forked-1idw1r

### Vant Version

4.3.0

### Describe the Bug

When update new version for Vant 4.3.0 and terminal will show the error message:
 ERR_PNPM_NO_MATCHING_VERSION_INSIDE_WORKSPACE  In : No matching version found for @vant/popperjs@* inside the workspace 

### Reproduce Steps

update vant version to 4.3.0

**Comment by codemariosec** [comment]
I also have this problem.

**Comment by zhoufenfens** [comment]
similar errors
```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:^
```

**Comment by leessmin** [comment]
Mayday Mayday Mayday，I also have this problem,

**Comment by waney** [comment]
I also have this problem.

**Comment by chenjiahan** [comment]
Sorry for the error, this bug has been fixed in vant@4.3.1.

The workspace protocol is used for monorepo development, and should be replaced before publishing. But it is published to npm and caused the bug.
